openslides/core/access_permissions.py

```diff
@@ -106,4 +105,8 @@ def get_full_data(self, instance):
 
         # Attention: The format of this response has to be the same as in
         # the retrieve method of ConfigViewSet.
+        if isinstance(instance, dict):
```

Comment:
I don't see the difference. You code does exactly the same. If you like, you can change it.